src/static/css/hint.css

```diff
@@ -0,0 +1,40 @@
+@import url('../themes/default/content.css');
+@import url('../themes/dark/content.css');
+@import url('../themes/greenmat/content.css');
+
+span.TridactylHint {
+    position: absolute !important;
+    font-family: var(--tridactyl-hintspan-font-family) !important;
+    font-size: var(--tridactyl-hintspan-font-size) !important;
+    font-weight: var(--tridactyl-hintspan-font-weight) !important;
+    text-transform: uppercase !important;
+    color: var(--tridactyl-hintspan-fg) !important;
+    background-color: var(--tridactyl-hintspan-bg) !important;
+    border-color: var(--tridactyl-hintspan-border-color) !important;
+    border-width: var(--tridactyl-hintspan-border-width) !important;
+    min-width: .75em !important;
+    border-style: var(--tridactyl-hintspan-border-style) !important;
+    padding: 0 1pt !important;
+    text-align: center !important;
+    z-index: 2147483647 !important;
+}
+
+.TridactylHintElem, .TridactylHintActive {
+    color: var(--tridactyl-hint-active-fg) !important !important;
```

Comment:
This is actually the font colour chosen for both the selected and unselected hints.